src/components/ha-chip-set.ts

```diff
@@ -8,50 +8,23 @@ import {
   TemplateResult,
   unsafeCSS,
 } from "lit";
-import { customElement, property } from "lit/decorators";
-import { fireEvent } from "../common/dom/fire_event";
-import "./ha-chip";
-
-declare global {
-  // for fire event
-  interface HASSDomEvents {
-    "chip-clicked": { index: string };
-  }
-}
+import { customElement } from "lit/decorators";
 
 @customElement("ha-chip-set")
 export class HaChipSet extends LitElement {
-  @property() public items = [];
-
   protected render(): TemplateResult {
-    if (this.items.length === 0) {
-      return html``;
-    }
     return html`
       <div class="mdc-chip-set">
-        ${this.items.map(
-          (item, idx) =>
-            html`
-              <ha-chip .index=${idx} @click=${this._handleClick}>
-                ${item}
-              </ha-chip>
-            `
-        )}
+        <slot></slot>
       </div>
     `;
   }
 
-  private _handleClick(ev): void {
-    fireEvent(this, "chip-clicked", {
-      index: ev.currentTarget.index,
-    });
-  }
-
   static get styles(): CSSResultGroup {
     return css`
       ${unsafeCSS(chipStyles)}
 
-      ha-chip {
+      slot::slotted(ha-chip) {
         margin: 4px;
```

Comment:
![image](https://user-images.githubusercontent.com/1444314/142691696-2a42cc6e-d4c6-4312-9dc5-a1dc330db663.png)

We now have on the left 8px (4px padding, 4px margin)

Docs show no margin on the left. Maybe we should do a margin-left: -4px on :host for chip-set? 
https://m3.material.io/components/chips/overview

Reply:
That is not new.